framework/test/randoop_single_coverage.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+################################################################################
+#
+# This script generates coverage data for Randoop generated tests over a single defects4j test.
+#
+################################################################################
+# Import helper subroutines and variables, and init Defects4J
+source test.include
+init
+
+# change 0 to 1 for extra logging info and to save temp files
+debug=0
+
+# master coverage file
+master_coverage=$TMP_DIR/coverage
+
+# Directory for Randoop test suites
+randoop_dir=$TMP_DIR/randoop
+
+# Generate tests for Lang-4
+# Mockito #1 and #3 don't work; problem finding classes in byte-buddy?
+# Thus the strange hack below
+projects=( Lang )
+type=f
+
+# Test suite source and number
+suite_src=randoop
+suite_num=1
+
+# probably should be a flag whether or not to keep existing data for cumlative run(s)
+#rm -f $master_coverage
+
+for pid in "${projects[@]}"; do
+    for bid in 4; do
```

Comment:
There is a fair amount of overlap between the two scripts `randoop_single_coverage.sh` and `randoop_coverage.sh`. Can we use one parameterized script?

The key difference is running Randoop on one vs. multiple project versions. Just an idea: `bid` could iterate over an array (similar to `pid`) and the content of that array could be defined based on the use case (single bid vs. multiple bids).